gen/src/write.rs

```diff
@@ -56,6 +57,9 @@ pub(super) fn gen(
                     .push(efn);
             }
         }
+        if let Api::CxxType(enm) = api {
```

Comment:
`enm` should always refer to a crate::syntax::Enum. We should use `ety` for crate::syntax::ExternType.

Reply:
Ah, I didn't see those.  Thanks!